cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -150,6 +150,29 @@ def test_embedded_ansible_repository_crud(ansible_repository, wait_for_ansible):
     assert view.description.value == updated_description
 
 
+@pytest.mark.rhel_testing
+@pytest.mark.tier(1)
+def test_embedded_ansible_repository_invalid_url_crud(appliance, wait_for_ansible):
+    """
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/6h
+        tags: ansible_embed
+    """
+    repositories = appliance.collections.ansible_repositories
+    repository = repositories.create(
+        name=fauxfactory.gen_alpha(),
+        url='https://github.com/sbulage/invalid_repo_url.git',
+        description=fauxfactory.gen_alpha())
+    view = navigate_to(repository, "Details")
+    assert view.entities.summary("Properties").get_text_of("Status") == "failed"
+
+    if repository.exists:
+        repository.delete()
+
```

Comment:
@sbulage 
- Please check ```delete_if_exists``` for ```repository``` if [this is teardown](https://github.com/ManageIQ/integration_tests/pull/8665/files#diff-af2579649707b347ff0dbbe799963d07R172).
- For teardown, you can write ```request.addfinalizer(repository.delete_if_exists)``` above  ```assert```.

Reply:
It's a CRUD operation, so I don't think finalizer required .?

Reply:
please explain, why CRUD operations test does not require tear-down?

Reply:
Any how I am checking that if repository exists then delete. Even in the Finalizer we are checking similar approach. If that makes sense.

Reply:
I agree but if assertion gets failed at line no.170 then test will not check ```repository.exists```. 
So as per my understanding we should have teardown step above assertion.
What do you think @sbulage ?

Reply:
Added Finalizer.